jira/client.py

```diff
@@ -1694,47 +1694,92 @@ def group_members(self, group: str) -> OrderedDict:
                 "Group members is not implemented in Jira before version 6.0, upgrade the instance, if possible."
             )
 
-        params = {"groupname": group, "expand": "users"}
-        r = self._get_json("group", params=params)
-        size = r["users"]["size"]
-        end_index = r["users"]["end-index"]
-
-        while end_index < size - 1:
-            params = {
-                "groupname": group,
-                "expand": f"users[{end_index + 1}:{end_index + 50}]",
-            }
-            r2 = self._get_json("group", params=params)
-            for user in r2["users"]["items"]:
-                r["users"]["items"].append(user)
-            end_index = r2["users"]["end-index"]
+        if self._version < (10, 0, 0):
```

Comment:
thank you for your contribution. Could you remove some of the duplication out of the code please?

Also dont forget to rebase.

Thank you!